fees/cian-yieldlayer/index.ts

```diff
@@ -0,0 +1,99 @@
+import { CHAIN } from "../../helpers/chains"
+import * as sdk from "@defillama/sdk";
+
+//cian yield layer doesn't have factory, so we hardcode the vaults
+const cianVaults = {
+    [CHAIN.ETHEREUM]: {
+        // "ylstETH Scroll Vault": "0x3498fDed9C88Ae83b3BC6a302108F2da408e613b", this one doesn't have vaultParams and no fee getter, but curiously the website shows 10% fee 
+        "stETH Yield Layer": "0xB13aa2d0345b0439b064f26B82D8dCf3f508775d",
+        "rsETH Yield Layer": "0xd87a19fF681AE98BF10d2220D1AE3Fbd374ADE4e",
+        "BTCLST Yield Layer": "0x6c77bdE03952BbcB923815d90A73a7eD7EC895D1",
+        "FBTC Yield Layer": "0x8D76e7847dFbEA6e9F4C235CADF51586bA3560A2",
+        "ezETH Yield Layer": "0x3D086B688D7c0362BE4f9600d626f622792c4a20",
+        "uniBTC Yield Layer": "0xcc7E6dE27DdF225E24E8652F62101Dab4656E20A",
+        "cbBTC Yield Layer": "0x7BA7c46e9F44d93AEF0Ddd37b80134438f60e15e",
+        "pumpBTC Yield Layer": "0xd4Cc9b31e9eF33E392FF2f81AD52BE8523e0993b",
+        // "LRT Yield Layer - steakLRT": "0x30282284Fc5290a049427ca01bB81A331A9F8107",
+        // "LRT Yield Layer - rstETH": "0x7CB6Bb4320622B85b265f596a4AD9f7Cc85F8797", these two doesn't have verified contract so we skip them
+        "fBTC Yield Layer Ⅲ": "0x9fdDAD44eD6b77e6777dC1b16ee4FCcCBaF0A019"
+    },
+    // [CHAIN.ARBITRUM]: {
+    //     "rsETH Yield Layer": "0x15cbFF12d53e7BdE3f1618844CaaEf99b2836d2A",
+    // },
+    // [CHAIN.SEI]: {
+    //     "redBTC Yield Layer": "0xff6771a9565F18638faB2972BA7Fc798ad8bCad0"
+    // }
+}
+
+const cianVaultABI = {
+    getVaultParams: "function getVaultParams() view returns (tuple(address underlyingToken, string name, string symbol, uint256 marketCapacity, uint256 managementFeeRate, uint256 managementFeeClaimPeriod, uint256 maxPriceUpdatePeriod, uint256 revenueRate, uint256 exitFeeRate, address admin, address rebalancer, address feeReceiver, address redeemOperator))",
+    totalAssets: "function totalAssets() view returns (uint256)"
+}
+
+const fetch = async ({createBalances, api, fromApi, toApi, chain}: FetchOptions) => {
+    const dailyFees = createBalances()
+    // call the vaults
+    const vaults = Object.values(cianVaults[chain]) as string[]
+    
+    const vaultsParams = (await api.multiCall({
+        abi: cianVaultABI.getVaultParams,
+        calls: vaults,
+        permitFailure: true
+    })).map((params) => ({
+        token: params.underlyingToken,
+        managementFeeRate: params.managementFeeRate,
+        revenueRate: params.revenueRate
+    }))
+
+    const vaultsTotalAssets = await fromApi.multiCall({
+        abi: cianVaultABI.totalAssets,
+        calls: vaults
+    })
+
+    const vaultsTotalAssetsEnd = await toApi.multiCall({
+        abi: cianVaultABI.totalAssets,
+        calls: vaults
+    })
+
+    vaultsTotalAssetsEnd.forEach((totalAssets, i) => {
+        dailyFees.add(vaultsParams[i].token, (totalAssets - vaultsTotalAssets[i]) * vaultsParams[i].managementFeeRate / 1e4)
```

Comment:
hmm, this is not necesarrily true? what if this is a big deposit or withdrawal in this time frame